components/dialog/stories/dialog.stories.js

```diff
@@ -170,7 +173,7 @@ export default {
 		status: {
 			type: "migrated",
 		},
-		layout: "padded",
+		layout: "fullscreen",
```

Comment:
Changing this `layout` to fullscreen mimics `main` and `s2-foundations-redux` a little better. I'm still not certain if this is a long term fix or not, but they certain look a little better (until the screen gets reeeeaally large).